docs/cep/query-guide/functions/time/dateSub.md

```diff
@@ -50,7 +50,7 @@ This query subtracts one hour from the timestamp value '1415715824000L' (in mill
 
 ```sql
 CREATE STREAM InputStream (inputDate string, unit string, amount int, inputFormat string);
-CREATE STREAM OutputStream (newDate string);
+CREATE SINK STREAM OutputStream (newDate string);
```

Comment:
In this example, Here we are taking "unit" as input from user and passing it into here:
"SELECT time:dateSub(inputDate, amount, unit, inputFormat) AS newDate"
BUT time:dateSub needs "unit" to be static value, it can't be dynamic, it has to be a static value for example, `MINUTE`, `HOUR`, `MONTH`, `YEAR`, `QUARTER`, `WEEK`, `DAY`, `SECOND`..

So it needs to be something like :
"SELECT time:dateSub(inputDate, amount, 'HOUR', inputFormat) AS newDate" (instead of 'HOUR' you can use any of the above but it has to be static)

Also, the explanation below need to be changed accordingly